src/app/pcasts/dao/discover_dao.py

```diff
@@ -15,9 +16,8 @@ def request_podcast_ml(url):
 
 def get_series_for_topic(topic_id, user_id, offset, max_search):
   if config.ML_ENABLED:
-    response = request_podcast_ml('/api/v1/series/topic/{}/?offset={}&max={}'
```

Comment:
where are you specifying `'/api/v1/series/topic/`

Reply:
It's not used anymore, this PR moves the non-ML series for topic recommendations to podcast-backend